bpmn-statistic/src/com/axonivy/utils/bpmnstatistic/bo/Node.java

```diff
@@ -1,14 +1,18 @@
 package com.axonivy.utils.bpmnstatistic.bo;
 
 import com.axonivy.utils.bpmnstatistic.enums.NodeType;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 
 public class Node {
   private NodeType type;
   private String id;
+  @JsonIgnore
   private String label;
-  private int labelValue;
+  private String labelValue;
```

Comment:
Did you inform PO abt this change? Reto as us to change the label value to int before